src/markdown-pages/contribute-to-quickstarts/instrument-library/send-events.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Send Events from Your Library'
+template: 'GuideTemplate'
+description: 'With our Events API you can ingest your data as the first step towards your quickstart contribution.'
+tags:
+  - quickstart
+---
+
+<Intro>
+
+This guide will walk you through how to instrument your application to send event data into the New Relic's [Event API](https://docs.newrelic.com/docs/data-apis/ingest-apis/introduction-event-api/). The Event API allow you to send events to New Relic without a need for an agent. First, let's cover what events are. An event is a record of something happening somewhere. What triggers the event and what data is captured is up to you to customize, which we'll discuss next. 
+
+</Intro>
+
+## Capture Events in your Library
```

Comment:
I'm confused why you have this section. It talks about a few things:

- That events have attributes (this is important)
- That events have an event-type attribute (this doesn't seem important, considering none of your examples later on set or make use of this attribute)
- That events can be queried on their event-type attribute (this really doesn't seem important in this document)

It seems like you could bring the first point up into the introduction and drop the rest of this section to me. Thoughts?